docs/integrations/ml-dist.rst

```diff
@@ -0,0 +1,512 @@
+======================================
+Distributed Deep-Learning with CrateDB
+======================================
+
+Abstract
+========
+Using Deep-Learning algorithms for Machine Learning Use-Cases becomes
+more and more common in the world of data science. A common library used
+for solving Deep-Learning problems is `TensorFlow`_, which will be used to
+predict the status of a water pump. CrateDB can easily be integrated into
```

Comment:
In the Abstract section, we usually try to explain how to integrate CrateDB with the tools without mentioning any use cases, in this case the water pump. Maybe something like,

```suggestion
for solving Deep-Learning problems is `TensorFlow`_, can be used to generate
predictions based on various machine data features. CrateDB can easily be integrated into
```